src/sentry_plugins/asana/plugin.py

```diff
@@ -202,7 +204,11 @@ def get_config(self, *args, **kwargs):
             client = self.get_client(user)
         except PluginIdentityRequired as e:
             self.raise_error(e)
-        workspaces = client.get_workspaces()
+        try:
+            workspaces = client.get_workspaces()
+        except HTTPError as e:
+            if e.response.status_code == 400:
```

Comment:
@ceorourke I think the check on the `url` you had before made sense because there could be other `400` errors that might happen.